File: java/sprykerFileCreator/model/generator/DocBlockGeneratorInterface.java
package sprykerFileCreator.model.generator;

import pav.sprykerFileCreator.model.renderer.dto.DocBlockItem;
import pav.sprykerFileCreator.model.renderer.dto.PhpClassInterface;

import java.util.List;

public interface DocBlockGeneratorInterface {

    List<DocBlockItem> getDocBlockItems(
            String[] classTypes,
            String bundleName
    ) throws Exception;

}

File: src/main/kotlin/com/github/tobi812/sprykerplugin/services/generator/DocBlockGeneratorInterface.kt
package com.github.tobi812.sprykerplugin.services.generator

import com.github.tobi812.sprykerplugin.services.renderer.dto.DocBlockItem
import java.lang.Exception

interface DocBlockGeneratorInterface {
    @Throws(Exception::class)
    fun getDocBlockItems(
        classTypes: Array<String>,
        bundleName: String
    ): List<DocBlockItem>
}
